torchtitan/models/norms.py

```diff
@@ -6,12 +6,17 @@
 
 import math
 
+from functools import partial
+
 import torch
 import torch.nn as nn
 
 import triton
 import triton.language as tl
 
+from torch.distributed._tensor.experimental import local_map
+from torch.distributed._tensor.placement_types import _Partial, Replicate, Shard
```

Comment:
weird, this failed in `CPU test` in CI but succeeded in `GPU Integration Test`. Is this because those tests are using different pytorch version and the CPU test one doesn't have the `Partial` publish PR in? Shall I keep `from torch.distributed._tensor.placement_types import Partial, Replicate, Shard` for now and change it to `from torch.distributed._tensor import Partial, Replicate, Shard` when CPU test can pass?

Reply:
I think the CPU test does not install nightly version of pytorch correctly, I'm not sure if this would help or not https://github.com/pytorch/torchtitan/pull/396

Reply:
doesn't seem right. 

CPU Test:
<img width="1366" alt="image" src="https://github.com/pytorch/torchtitan/assets/12968408/df9a856f-681f-4c2b-9794-93c85da5114b">

GPU Integration Test:
<img width="1610" alt="image" src="https://github.com/pytorch/torchtitan/assets/12968408/9dcebdf1-2b62-4a75-86af-d99579503a43">